.changeset/brave-timers-dream.md

```diff
@@ -0,0 +1,7 @@
+---
+"@aws-amplify/ui-docs": patch
+"@aws-amplify/ui-react": patch
+"@aws-amplify/ui": patch
+---
+
+added new isMultiple and selectSize props
```

Comment:
Can we add an example usage and call out this is for SelectField?

Reply:
Good call, updated